backend/backend.py

```diff
@@ -229,9 +229,17 @@ def send_recording_error(self) -> None:
         self.publish_recording_status_func(dumps(message))
         print(format_exc())
 
-    def send_video_status(self) -> None:
+    def send_video_status(self, status: bool = None) -> None:
         """ Publish the camera's status to the camera's online topic. """
-        self.publish_video_status_func(dumps({"online": self._is_video_on()}))
+        self.publish_video_status_func(
+            dumps(
+                {
+                    "online": status
+                    if status is not None
+                    else self._is_video_on()
```

Comment:
I hate to hold up this PR, but just one thing - I find this bit of code really hard to read with black's formatting. Could we instead add
```python
if status is None:
    status = self._is_video_on()
```
and then just use `status`?

Reply:
Done
31b39e3